src/flash/flasher.cpp

```diff
@@ -104,7 +104,7 @@ void Flasher::flash()
     static QString cmd = QStringLiteral("\"%0\" -w \"%1\" %2 -g 0x0 -b %3 %4").arg(
                              stm32flashPath(),
                              firmwareFileInfo.absoluteFilePath(),
-                             _verify ? "-v" : "",
+                             _verify ? QStringLiteral("-v") : QString(),
```

Comment:
What is the reasoning here? Is this potentially a functional change?

Reply:
Same behavior. QStringLiteral() will transform the strings from utf8 or ascii to utf16 in compile time, and "" will produce a empty string (just with an \0 byte) that will not be appended anywhere because it's empty, while QString() will produce.. nothing. not even a \0 byte. it will not try to convert ascii to utf-16 and thus will execute faster.